index.html

```diff
@@ -0,0 +1,635 @@
+                                </li>
+                                <li>
+                                    <img class="icon-x-small" src="/images/twitter/tips.png" alt="tip">
+                                    <p><span class="style-two"><em>Tip</em></span></p>
+                                </li>
+                            </ul> 
+                        </li>
+                        <li class="tweet three-rows__expand-two dashed-border-pink">
+                            <img class="icon-x-small retweet-icon" src="/images/twitter/retweet-grey-bold.png" alt="retweet-bold">
+                            <h4>Hera Retweeted</h4>
+                            <img class="profile-pic-small" src="/images/twitter/Zeus.png" alt="profile-pic-small">
+                            <div class="tweet-content two-rows__expand-two">
+                                <div>
+                                    <h6>Zeus</h6>
+                                    <p><span class="style-two">@zues - 45m</span></p>
+                                </div>
+                                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Odio quas architecto assumenda voluptas. Optio suscipit cumque aperiam est atque aliquid impedit enim natus eum ad, exercitationem deserunt quisquam excepturi perspiciatis!</p>
+                            </div>
+                            <img class="icon-small" src="/images/twitter/three-dots-grey.png" alt="three-dots">
+                            <ul class="tweet-icon-holder">
+                                <li>
+                                    <img class="icon-x-small" src="/images/twitter/speech.png" alt="speech">
+                                    <p><span class="style-two"></span></p>
+                                </li>
+                                <li>
+                                    <img class="icon-x-small" src="/images/twitter/retweet-grey.png" alt="retweet">
+                                    <p><span class="style-two">3</span></p>
+                                </li>
+                                <li>
+                                    <img class="icon-x-small" src="/images/twitter/heart.png" alt="heart">
+                                    <p><span class="style-two">22</span></p>
+                                </li>
+                                <li>
+                                    <img class="icon-x-small" src="/images/twitter/inbox.png" alt="inbox">
+                                    <p><span class="style-two"></span></p>
+                                </li>
+                                <li>
+                                    <img class="icon-x-small" src="/images/twitter/tips.png" alt="tip">
+                                    <p><span class="style-two"><em>Tip</em></span></p>
+                                </li>
+                            </ul> 
+                        </li>
+                    </ul> 
+                </div>
```

Comment:
could use <main> "" </main> instead of a div for better readability for your feed/main section

Reply:
could add some spacing as well as to better distinguish between the different sections/areas of the site

Reply:
agreed, when I was reading other peoples code it was easier when there where regular line spaces to separate sections. will implement this into my project